### ISSUE TITLE
Feature request: Charity Leaderboard

### ISSUE THREAD

**Issue by ahalekelly** [manifoldmarkets/manifold#534]
Incentivize charity donations by making a leaderboard for it

**Comment by akrolsmir** [comment]
Yup, love this idea in general. A few things to think about:

- There are two kinds of leaderboards that could make sense: one for total donations per user, and another per charity. The latter is a nice kind of competitive framing to cheer on your preferred charity (although, it's already kind of what the /charity page encourages). 

- A third leaderboard that could make the most sense, actually, is the per-user per-charity leaderboard. This encourages large single donations to one charity as opposed to splitting up across multiple charities, which is a good countervailing pressure to what the quadratic funding match mechanism does.

- Where should this leaderboard go? Could be either on /leaderboards or /charity...

**Comment by ahalekelly** [comment]
Yeah I'm thinking put a per user leaderboard on /leaderboards, to signify that charity is a core part of Manifold and not an afterthought or just marketing

**Comment by PatMyron** [comment]
Largest non-employee donations so far:

[JoelBecker](https://manifold.markets/JoelBecker) to [Animal Welfare](https://manifold.markets/charity/animal-welfare-fund)/[Givewell](https://manifold.markets/charity/givewell-maximum-impact-fund)
[Preen](https://manifold.markets/Preen) to [LTFF](https://manifold.markets/charity/long-term-future-fund)
